infra-parent/infra-core/src/main/java/com/redhat/service/smartevents/infra/core/exceptions/mappers/JsonMappingExceptionMapper.java

```diff
@@ -25,14 +27,14 @@ protected JsonMappingExceptionMapper() {
         //CDI proxy
     }
 
-    public JsonMappingExceptionMapper(BridgeErrorService bridgeErrorService) {
-        super(bridgeErrorService, ExternalUserException.class);
+    public JsonMappingExceptionMapper(BridgeErrorService bridgeErrorService, Instance<ErrorHrefVersionProvider> builders) {
+        super(bridgeErrorService, ExternalUserException.class, builders);
     }
 
     @Override
     public Response toResponse(JsonMappingException e) {
-        LOGGER.debug("Failure", e);
         Throwable cause = Objects.nonNull(e.getCause()) ? e.getCause() : e;
+        // TODO: refactor, add tests
```

Comment:
Is this still relevant? You said on gChat that you'd done all tests.

Reply:
yup, I forgot to remove the comment :+1: